NuRadioMC/test/atmospheric_Aeff/config.yaml

```diff
@@ -0,0 +1,15 @@
+noise: False  # specify if simulation should be run with or without noise
+sampling_rate: 2.  # sampling rate in GHz used internally in the simulation.
+speedup:
+  minimum_weight_cut: 1.e-5
+  delta_C_cut: 0.698  # 40 degree
+  redo_raytracing: True  # redo ray tracing even if previous calculated ray tracing solutions are present
+  distance_cut: True # Skip ray tracing if vertex distance is greater than a distance cut that is a function of shower energy.
+propagation:
+  ice_model: southpole_2015
+signal:
+  model: Alvarez2000
+trigger:
+  noise_temperature: 300  # in Kelvin
```

Comment:
to increase statistics you could also lower the noise temperature. But this is equivalent to just lowering the threshold. As we don't add noise, maybe this would be clever to reduce computing time and increase precision. But then the test is testing unrealistically low signals... so I'm not sure if it will be a good choice in the end

Reply:
Another way is to remove pair production.